crates/ironrdp-rdpdr/src/pdu/efs.rs

```diff
@@ -1615,6 +1628,8 @@ bitflags! {
         const FILE_ATTRIBUTE_PINNED = 0x00080000;
         const FILE_ATTRIBUTE_UNPINNED = 0x00100000;
         const FILE_ATTRIBUTE_RECALL_ON_DATA_ACCESS = 0x00400000;
+
+        const _ = !0;
```

Comment:
It just occurred to me this is the culprit. The `Cargo.lock` in the `fuzz/` folder is not updated to `bitflags v2.4`. You need to specify this in the `Cargo.toml`:

```rust
bitflags = "2.4"
```

Reply:
We were using the same `bitflags = "2"` throughout, so I added `bitflags = "2.4"` to the workspace root Cargo.toml and updated all of the `Cargo.lock`s

1e5d8e11656cd7c2ff0eb7baba694a0ba4372e5f